tidb-resource-control.md

```diff
@@ -0,0 +1,150 @@
+
+> **Note:**
+>
+> - When you bind a user to a resource group by using `CREATE USER` or `ALTER USER`, it will not take effect for the user's existing sessions, but only for the user's new sessions.
+> - If a user is not bound to a resource group or is bound to a `default` resource group, the user's requests are not subject to TiDB's flow control restrictions. The `default` resource group is currently not visible to the user and cannot be created or modified.
+
+### Step 1. Enable the resource control feature
+
+Enable flow control for the resource group.
+
+```sql
+SET GLOBAL tidb_enable_resource_control = 'ON';
+```
+
+<CustomContent platform="tidb-cloud">
+
+In TiKV, set the parameter `resource_control.enabled` to `true`. The parameter `resource_control.enabled` is disabled by default. You need to contact the [PingCAP support team](/tidb-cloud/tidb-cloud-support.md) to enable it.
+
+</CustomContent> 
+
+### Step 2. Create a resource group, and then bind users to it
+
+The following is an example of how to create a resource group and bind users to it.
+
+1. Create a resource group `rg1`. The RU backfill rate is 500 RU per second and allows applications in this resource group to overrun resources.
+
+    ```sql
+    CREATE RESOURCE GROUP IF NOT EXISTS rg1 RU_PER_SEC = 500 BURSTABLE;
+    ```
+
+2. Create a resource group `rg2`. The RU backfill rate is 500 RU per second and does not allow applications in this resource group to overrun resources.
+
+    ```sql
+    CREATE RESOURCE GROUP IF NOT EXISTS rg2 RU_PER_SEC = 600;
+    ```
+
+3. Bind users `usr1` and `usr2` to resource groups `rg1` and `rg2` respectively.
+
+    ```sql
+    ALTER USER usr1 RESOURCE GROUP rg1;
+    ```
+
+    ```sql
+    ALTER USER usr2 RESOURCE GROUP rg2;
+    ```
+
+After you complete the above operations of creating resource groups and binding users, the resource consumption of newly created sessions will be controlled by the specified quota. If the system load is relatively high and there is no spare capacity, the resource consumption rate of `usr2` will be strictly controlled not to exceed the quota. Because `usr1` bound by `rg1` is configured with `BURSTABLE`, the consumption rate of `usr1` is allowed to exceed the quota.
+
+If the resource group corresponds to a request with insufficient quota, the client's request will wait. If the wait time is too long, the request will report an error.
+
+## Monitoring and charts
+
+TiDB regularly collects runtime information about resource control and provides visual charts of the metrics in Grafana's **TiDB** > **Resource Control** panel. The metrics are detailed in the **Resource Control** section of [TiDB Important Monitoring Metrics](/grafana-tidb-dashboard.md).
+
+## Tool Compatibility
+
+The resource control feature is not compatible with data export/import and replication tools including BR, TiDB Lightning, and TiCDC.
```

Comment:
This needs more explanation.

- Can I still export data?
- Are the resource groups gone after a backup/restore operation?
- What happens when running two TiDB clusters with TiCDC, do the resource groups and/or statements that create resource groups replicate to the secondary cluster?
- What happens with TiDB to MySQL replication with TiCDC? Does it cause syntax errors on the target?
- Will tools like BR and Dumpling give warnings for when resource groups are enabled?

Reply:
resource group meta information will be ignored by those tools. No impact to normal user data.